x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/parser/StatementParserTests.java

```diff
@@ -3112,4 +3114,42 @@ public void testFieldNamesAsCommands() throws Exception {
     static Alias alias(String name, Expression value) {
         return new Alias(EMPTY, name, value);
     }
+
+    public void testValidRrf() {
```

Comment:
maybe add some negative tests, say, for if RRF is used without a FORK, etc.

Reply:
The parser will not complain if we have invalid queries using RRF.
A query `FROM search-movies | RRF` will not produce parser errors.
To test for invalid queries, I added tests in `AnalyzerTests` - see `testRrfError`.